UVAtlas/isochart/isochartengine.cpp

```diff
@@ -1803,15 +1814,21 @@ HRESULT CIsochartEngine::TryEnterExclusiveSection()
     {
         return E_ABORT;
     }
+#else
+    return m_mutex.try_lock() ? S_OK : E_ABORT;
```

Comment:
Is there a reason to have divergent code here and not just use the STL mutex everywhere?

Reply:
``mutex`` isn't supported for STL_VER=110 scenarios, but that may be less of an issue these days...